### PR TITLE
Add a link to the API documentation.

### PR DESCRIPTION
**Pull request by hawkfalcon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +148,9 @@ docker run --rm -it -v "$(pwd)":/paste jkcclemens/paste
 By default, that will build paste, but you can also append `/bin/bash` to the command to start a
 Bash shell in the working environment.
 
+## API
+See the [paste API documentation](api.md)
```
Comment (jkcclemens) [inline README.md]: Can I get a period at the end? <3

Comment (mbax) [reply README.md]: I don't know about that. Should an exclamation mark be considered?

Comment (mbax) [reply README.md]: Perhaps confuse the users instead with an interrobang‽

Comment (Zarthus) [reply README.md]: I think we should reword it to be less demanding, maybe "If you would like to see the API docs, please have a look at [...]". Maybe they don't really care about it.

Comment (mbax) [reply README.md]: "If you wish to read the API documentation, you are invited to view it [here](api.md) but should not feel any pressure to do so. You are wonderful just the way you are."

Comment (hawkfalcon) [reply README.md]: It's really missing a heart, @mbax

Comment (mbax) [reply README.md]: OMG HE RESPONDED TO ME :heart:

Comment (lol768) [reply README.md]: >Can I get a period at the end? <3

They're called full-stops
